pages/common/az-logout.md

```diff
@@ -0,0 +1,12 @@
+# az logout
+
+> Logout from an Azure subscriptions.
```

Comment:
```suggestion
> Log out from an Azure subscriptions.
```
I think that description may be a bit misleading. The docs say `Log out to remove access to Azure subscriptions.`, while this can be interpreted as `Cancel an Azure subscription`.

Reply:
Sure, so do we think we should explicitly specify it does not `Cancel` only `Logs Out` ?

Reply:
Isn't that a big difference? `Log out` means you just can't access it right now and `Cancel` means it's gone.

Reply:
Yes, `Cancel` & `Log Out` are different things, i actually now got what you meant. The official documentation was misleading :)